debug-plugin.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+PLUGIN_ID=$(build/bin/manifest id)
+
+if [[ -z ${PLUGIN_ID} ]]
```

Comment:
I /think/ `set -o pipefail` above might do this for free, though I'm not sure.

Reply:
Yah, I'm not sure either... I'll leave it, unless you have a strong opinion (then I'll investigate and test).

Reply:
It would be great to simplify if possible! (Reduces overall cognitive load when reading.)